scripts/validation/embed-logs/Components Object-json-format.json

```diff
@@ -0,0 +1,787 @@
+        }
+      }
+    },
+    "userSignupReply": {
+      "address": "users.signup.reply",
+      "description": "Channel for user signup replies",
+      "messages": {
+        "userSignedUpReply": {
+          "summary": "User signup reply message",
+          "payload": {
+            "type": "object",
+            "properties": {
+              "status": {
+                "type": "string",
+                "description": "Status of the signup process"
+              },
+              "message": {
+                "type": "string",
+                "description": "Additional information"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "operations": {
+    "sendUserSignUp": {
+      "action": "send",
+      "title": "User sign up",
+      "summary": "Action to sign a user up.",
+      "description": "A longer description",
+      "channel": {
+        "$ref": "#/channels/user"
+      },
+      "security": [
+        {
+          "type": "oauth2",
+          "description": "The oauth security descriptions",
+          "flows": {
+            "clientCredentials": {
+              "tokenUrl": "https://example.com/api/oauth/dialog",
+              "availableScopes": {
+                "subscribe:auth_revocations": "Scope required for authorization revocation topic"
+              }
+            }
+          },
+          "scopes": [
+            "subscribe:auth_revocations"
+          ]
+        }
+      ],
+      "tags": [
+        {
+          "name": "user"
+        },
+        {
+          "name": "signup"
+        },
+        {
+          "name": "register"
+        }
+      ],
+      "bindings": {
+        "amqp": {
+          "ack": false
+        }
+      },
+      "messages": [
+        {
+          "$ref": "#/channels/user/messages/userSignedUp"
+        }
+      ],
+      "reply": {
+        "address": {
+          "location": "$message.header#/replyTo"
+        },
+        "channel": {
+          "$ref": "#/channels/userSignupReply"
+        },
+        "messages": [
+          {
+            "$ref": "#/channels/userSignupReply/messages/userSignedUpReply"
+          }
+        ]
+      },
+      "traits": []
+    }
+  },
+  "components": {
+    "schemas": {
+      "Category": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "name": {
+            "type": "string"
+          }
+        }
+      },
+      "AvroExample": {
+        "schemaFormat": "application/vnd.apache.avro+json;version=1.9.0",
+        "schema": {
+          "type": "record",
+          "name": "UserCreate",
+          "namespace": "com.example",
+          "fields": [
+            {
+              "name": "id",
+              "type": "int"
+            },
+            {
+              "name": "name",
+              "type": "string"
+            }
+          ],
+          "$ref": "path/to/user-create.avsc#/UserCreate"
```

Comment:
Error for this file:
```bash
Warning in Components Object-json-format: "AvroExample" property must match "then" schema
Warning in Components Object-json-format: "schema" property must match exactly one schema in oneOf
Warning in Components Object-json-format: ENOENT: no such file or directory, open 'path/to/user-create.avsc'
```

@smoya Potential fix?

Reply:
https://github.com/asyncapi/spec/pull/1059#discussion_r1665911946

Reply:
Got it. Working on it...

Reply:
Same issue as https://github.com/asyncapi/spec/pull/1059#discussion_r1666331157

Marking resolved